frontend/src/components/RegistrationForm.js

```diff
@@ -9,6 +11,34 @@ export default function RegistrationForm() {
 
     async function signUp(event) {
         event.preventDefault();
+
+        try {
+            const response = await axios.post(
+                "http://localhost:8000/api/register",
+                {
+                    name,
+                    email,
+                    password,
+                }
+            );
+
+            setName("");
+            setEmail("");
+            setPassword("");
+            setConfirmPassword("");
+            toast.success(response.data.message, {
+                position: "top-right",
+                autoClose: 5000,
+                hideProgressBar: false,
+                closeOnClick: true,
+                pauseOnHover: true,
+                draggable: true,
+                progress: undefined,
+                theme: "light",
+            });
+        } catch (e) {
```

Comment:
dont just use console log on error. Create a toastify error if you have validation error from BE api.
Also, you dont have FE validation, pls create FE validation.

Reply:
@elyric-sun-asterisk-dev toastify is done.
Left to do is to add validation error before sending it to backend.
So it wont be able to send any request to BE if there is a validation error.
You can use any package for it and find the right package that suits in your app.